--- build.SickBeard.git/lib/unrar2/unix.py	2015-09-10 00:05:50.001568917 -0500
+++ patches/unix.py	2015-09-29 20:25:19.844265864 -0500
@@ -160,7 +160,7 @@
                     data['size'] = int(fields[0])
                     attr = fields[5]
                     data['isdir'] = 'd' in attr.lower()
-                    data['datetime'] = time.strptime(fields[3]+" "+fields[4], '%d-%m-%y %H:%M')
+                    data['datetime'] = time.strptime(fields[3]+" "+fields[4], '%Y-%m-%d %H:%M')
                     data['comment'] = None
                     data['volume'] = None
                     yield data
@@ -176,7 +176,7 @@
                 data['size'] = int(fields[1])
                 attr = fields[0]
                 data['isdir'] = 'd' in attr.lower()
-                data['datetime'] = time.strptime(fields[2]+" "+fields[3], '%d-%m-%y %H:%M')
+                data['datetime'] = time.strptime(fields[2]+" "+fields[3], '%Y-%m-%d %H:%M')
                 data['comment'] = None
                 data['volume'] = None
                 yield data
